js/hole-ng.js

```diff
@@ -176,25 +175,30 @@ const runCode = document.querySelector('#run a').onclick = async () => {
     status.style.background = data.Pass ? 'var(--green)' : 'var(--red)';
     statusH2.innerText      = data.Pass ? 'Pass 😀'      : 'Fail ☹️';
 
-    result = {
-        arguments: data.Argv.join('\n'),
-        diff:      data.Diff,
-        errors:    data.Err,
-        expected:  data.Exp,
-        output:    data.Out,
-    };
-
-    for (const div of statusDivs)
-        div.replaceChildren(new EditorView({
-            state: EditorState.create({
-                doc: result[div.id],
-                extensions: [
-                    ...baseExtensions,
-                    EditorView.editable.of(false),
-                    EditorView.lineWrapping,
-                    div.id == 'diff' ? extensions.diff : [],
-                ] }),
-        }).dom);
+    const argDiv = document.querySelector('#status > #arguments');
+    // Remove all arg spans
+    while (argDiv.firstChild)
+        argDiv.removeChild(argDiv.firstChild);
+
+    // Add a span for each arg
+    for(const arg of data.Argv) {
+        argDiv.appendChild(<span>{arg}</span>);
+        argDiv.appendChild(document.createTextNode(' '));
+    }
+
+    document.querySelector('#status > #diff').replaceChildren(new EditorView({
```

Comment:
I would hope `#diff` (and `#errors`, etc.) are all unique since they're IDs, we should need the extra `#status` selector.